src/structures/PlayerSummary.js

```diff
@@ -21,6 +21,10 @@ class PlayerSummary extends Player {
 		this.countryCode = player.loccountrycode;
 		this.stateCode = player.locstatecode;
 		this.cityID = player.loccityid;
+		this.gameserverip = player.gameserverip;
+        	this.gameserversteamid = player.gameserversteamid;
+        	this.gameextrainfo = player.gameextrainfo;
+        	this.gameid = player.gameid;
```

Comment:
I would prefer if these four properties were in a `this.game` object; furthermore, they should be camel cased.

Reply:
These properties directly relate to the player revealing which game and server that player is using currently